src/scripts/run-production-migration.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+
+/**
+ * Production Migration Runner for Mentoring Service
+ * Handles 30+ lakh records with Citus distribution
+ */
+
+require('dotenv').config({ path: '../.env' })
```

Comment:
@sumanvpacewisdom create a folder - tenantDataMigrations/script.js and helper.js